public/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <form action="/submit" method="POST" enctype="text/plain">
+    <label for="date">Enter date</label>
+    <input type="date" id="date" name="date" required>
+    
+    <label for="title">Enter title</label>
+    <input type="text" id="title" name="title" required>
+
+    <label for="amount">Enter amount</label>
+    <input type="number" id="amount" name="amount" required>
+
+    <button type="submit">Submit</button>
+  </form>
+</body>
+</html>
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)